sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollationExpressionSuite.scala

```diff
@@ -168,6 +168,7 @@ class CollationExpressionSuite extends SparkFunSuite with ExpressionEvalHelper {
   }
 
   test("CollationKey generates correct collation key for collated string") {
+    val b: Byte = 0x2B
```

Comment:
This is the same, the `CollationKey` is different. 
- In version `75.1`, its value is `0x2A` (`42`), 
- while in version `76.1`, its value is `0x2B` (`43`)

Reply:
This is also related to `CollationKey`.

Reply:
Could you add a comment before line 171, please, @panbingkun ?

Reply:
Sure